mailpoet/assets/js/src/automation/listing/store/selectors.ts

```diff
@@ -1,6 +1,11 @@
 import { State } from './types';
 import { Workflow } from '../workflow';
+import { workflowCount } from '../../config';
 
 export function getWorkflows(state: State): Workflow[] {
   return state.workflows;
 }
+
+export function getWorkflowCount(state: State): number {
+  return state.workflows ? state.workflows.length : workflowCount;
```

Comment:
Curious: Why do we need to "import" the workflowCount. To me it looks like `return state.workflows ? state.workflows.length : 0;` would do the same.

Reply:
Ah: I think this prevents the <Onboarding /> to show up while we just load the workflows.

Reply:
Yeah, the imported count tells you if you will have any workflows at all and you can decide to show either onboarding or a loader for the listing.

I was thinking to preload even the workflow list in the template but then I started to think about tabs and paging and stuff and just left the count as John did. In his version the count was a separate field in the state but it then creates too many states of "count" vs. "workflows" so I just used it in the selector when workflows are not loaded yet.